node/CHANGELOG.md

```diff
@@ -36,7 +36,7 @@ All notable changes to this project will be documented in this file.  The format
 * Connection handshake timeouts can now be configured via the `handshake_timeout` variable (they were hardcoded at 20 seconds before).
 * `Key::SystemContractRegistry` is now readable and can be queried via the RPC.
 * Requests for data from a peer are now de-prioritized over networking messages necessary for consensus and chain advancement.
-* JSON-RPC responses which fail to provide requested data will now also include an indication of that node's highest contiguous block height range, i.e. the block heights for which it holds all global state
+* JSON-RPC responses which fail to provide requested data will now also include an indication of that node's available block range, i.e. the block heights for which it holds all global state
```

Comment:
This does not mention how the contiguous sequence is calculated.

Reply:
Addressed in [e8486b1](https://github.com/casper-network/casper-node/pull/2789/commits/e8486b14819a33f282c2fbc0075a42c33af45db9).